### PR TITLE
docs: Add doc file regarding the Github conventions

### PR DESCRIPTION
**Pull request by JotaFan**

### CODE REVIEW

File: docs/conventions.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- github pages actions: *-to-pages
+- ro-crates: *-crate
+- web components: *-widget
+
+
+### Branches
+
+There should be two default branchs on each repo:
+- main (release ready code)
+- gh-pages (documentation) -> this branch is maintened automatically by a documentation workflow
```
Comment (laurianvm) [inline docs/conventions.md:26]: (spelling for 'maintained')

File: docs/conventions.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+## Rules
+
+### Pull Request
+
+A new PR from a new branch should be made on every code change. 
+- There is not direct push to main
+
+When opening a PR a name should be given (Can be like the branch name), and a description explaining the goal of the PR, and also using [GitHub keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) if available.
+
+To be able to merge a PR it need to:
```
Comment (laurianvm) [inline docs/conventions.md:75]: (spelling for 'needs'

File: docs/conventions.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Have at least one aproval
```
Comment (laurianvm) [inline docs/conventions.md:76]: (spelling for 'approval')

**Review by laurianvm — CHANGES_REQUESTED** [review]
+ one question: the checker actions that need to have run & passed to be able to merge the PR, are these run automatically?

**Comment by JotaFan** [comment]
> * one question: the checker actions that need to have run & passed to be able to merge the PR, are these run automatically?

Yes, they do. It's this githuc action:
https://gitlab.vliz.be/datac/templating/cookiecutter-py-module/-/blob/refactor/gh_convention/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/pr_validation.yml

**Review by cedricdcc — APPROVED** [review]

**Review by laurianvm — APPROVED** [review]

**Pull request merged**